error_reporting/tests/system.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+from google.cloud import error_reporting
+
+
+class TestErrorReporting(unittest.TestCase):
+
+    def test_report_exception(self):
+        client = error_reporting.Client()
+        try:
+            raise NameError
+        except NameError:
+            client.report_exception()
```

Comment:
Yes I prefer this one. A single test case is small enough to fit in the spirit of small PRs. You can add a skip ([e.g.][1]) so that the test only runs for gRPC.

[1]: https://github.com/GoogleCloudPlatform/google-cloud-python/blob/df30d1aa9fddacdc14f6c2412feb445e9d459e95/speech/tests/system.py#L212-L213